pom.xml

```diff
@@ -75,17 +75,17 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins.workflow</groupId>
         <artifactId>workflow-cps</artifactId>
-        <version>2.29</version>
+        <version>2.33-20170525.192309-1</version> <!-- TODO: update to release once 2.33 is released -->
       </dependency>
       <dependency>
         <groupId>org.jenkins-ci.plugins.workflow</groupId>
         <artifactId>workflow-job</artifactId>
-        <version>2.9</version>
+        <version>2.11</version>
```

Comment:
No problem, just making sure you did not forget to update dependencies to match the new upstream PRs.